vocode/streaming/action/dtmf.py

```diff
@@ -76,8 +77,20 @@ def __init__(self, action_config: DTMFVocodeActionConfig):
         )
 
     async def run(self, action_input: ActionInput[DTMFParameters]) -> ActionOutput[DTMFResponse]:
-        logger.error("DTMF not yet supported with Twilio")
+        buttons = action_input.params.buttons
+        keypad_entries: List[KeypadEntry]
+        try:
+            keypad_entries = [KeypadEntry(button) for button in buttons]
+        except ValueError:
+            logger.warning(f"Invalid DTMF buttons: {buttons}")
+            return ActionOutput(
+                action_type=action_input.action_config.type,
+                response=DTMFResponse(success=False),
```

Comment:
Have we tested these outputs are properly sent to the LLM? When I look at [action_result_to_string() for DTMF](https://github.com/vocodedev/vocode-core/blob/18761bb862f66cf8bda04724ee3d3ba319e7a98a/vocode/streaming/action/dtmf.py#L31) it only has a successful message. We should update it to say "invalid dtmf buttons" in this case

Reply:
updated + added test